storm-server/src/main/java/org/apache/storm/scheduler/resource/normalization/NormalizedResourceRequest.java

```diff
@@ -187,4 +187,29 @@ public double getTotalCpu() {
     public NormalizedResources getNormalizedResources() {
         return this.normalizedResources;
     }
+
+    public void removeOffHeap(final double offHeap) {
```

Comment:
where is this used?  Why increment?

Reply:
This is unused method. I was trying some thing else. Let me delete this method.